README.md

```diff
@@ -36,21 +36,21 @@ Get a list of skills
                 "title": "title1",
                 "description": "description1",
                 "examples": null,
-                "categoryId": "1"
+                "categoryTitle": "1"
```

Comment:
Wouldn't these be `category` objects, as of LFH-49?

Reply:
i think i'm going to do a bigger readme update on a following ticket if that's ok